bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/report/job/AnalyzeJob.java

```diff
@@ -42,51 +40,64 @@ private AnalyzeJob(String name) {
      */
     public AnalyzeJob(IProject project) {
         super("Running CodeChecker Analyze");
-        this.project = project;
-        ccec = new CodeCheckEnvironmentChecker(
-                CodeCheckerContext.getInstance().getCcProject(project));
+        this.project = CodeCheckerContext.getInstance().getCcProject(project);
+        config = this.project.getCurrentConfig();
     }
 
     @Override
     protected IStatus run(IProgressMonitor monitor) {
         Logger.log(IStatus.INFO, "Running AnalyzeJob");
-        
-        Optional<String> analyzeLog = moveLogFile();
-        if(analyzeLog.isPresent()) {
-            AtomicInteger taskCount = new AtomicInteger(0);
-            try {
-                //read the file
-                Files.lines(Paths.get(analyzeLog.get())).forEach(new Consumer<String>() {
+        try {
+            copyLogFile();
+        } catch (IOException e) {
+            Logger.log(IStatus.ERROR, "Couldn't copy logfile!");
+        }
 
-                    @Override
-                    public void accept(String t) {
-                        if ( t.contains("\"command\":")){
-                            taskCount.incrementAndGet();
-                        }
+        AtomicInteger taskCount = new AtomicInteger(0);
+
+        // read the file
+        try {
+            Files.lines(logFile).forEach(new Consumer<String>() {
+
+                @Override
+                public void accept(String t) {
+                    if (t.contains("\"command\":")) {
+                        taskCount.incrementAndGet();
                     }
-                });
-            } catch (IOException e) {
-                // isPresent() should render this unneeded. 
-            }
-            monitor.beginTask("Starting Analysis...", taskCount.get()*2);
-            ccec.processLog(analyzeLog.get(), true, monitor, taskCount.get()*2);
+                }
+            });
+        } catch (IOException e) {
+            Logger.log(IStatus.ERROR, "Couldn't read logFile!");
         }
+
+        monitor.beginTask("Starting Analysis...", taskCount.get() * 2);
+        config.getCodeChecker().analyze(logFile, true, monitor, taskCount.get() * 2, config);
```

Comment:
TODO There is a npe thrown at this line, when there is no CodeChecker configured.